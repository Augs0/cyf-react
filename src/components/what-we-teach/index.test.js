import React from 'react';
import { shallow } from 'enzyme';
import WhatWeTeach from './';

describe('WhatWeTeach', () => {
  it('renders the component', () => {
    expect(shallow(<WhatWeTeach />)).toMatchSnapshot();
  });
});
