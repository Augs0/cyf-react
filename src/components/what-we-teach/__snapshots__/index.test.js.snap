// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WhatWeTeach renders the component 1`] = `
<div
  className="section-container section-container-gray"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-6 col-md-6"
      >
        <h2>
          What we teach
        </h2>
        <div
          className="divider-1"
        >
          <span />
        </div>
        <br />
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-sm-5ths"
      >
        <div
          className="md-icon"
        >
          <img
            alt="Honeycomb"
            src="honeycomb.svg"
          />
        </div>
        <h4>
          Non-technical skills
        </h4>
        <p>
          Vital techniques and tools to make learning and working together easier and more effective.
        </p>
      </div>
      <div
        className="col-sm-5ths"
      >
        <div
          className="md-icon"
        >
          <img
            alt="Tablet"
            src="tablet.svg"
          />
        </div>
        <h4>
          HTML & CSS
        </h4>
        <p>
          Fundamentals of the web. Structure and style static web pages using best front-end practices.
        </p>
      </div>
      <div
        className="col-sm-5ths"
      >
        <div
          className="md-icon"
        >
          <img
            alt="Code"
            src="code.svg"
          />
        </div>
        <h4>
          JavaScript
        </h4>
        <p>
          A powerful modern scripting language to build web applications.
        </p>
      </div>
      <div
        className="col-sm-5ths"
      >
        <div
          className="md-icon"
        >
          <img
            alt="NodeJS"
            src="nodejs.svg"
          />
        </div>
        <h4>
          Node.js
        </h4>
        <p>
          Web application back-end development powered by modern and fast Node.js.
        </p>
      </div>
      <div
        className="col-sm-5ths"
      >
        <div
          className="md-icon"
        >
          <img
            alt="React"
            src="react.svg"
          />
        </div>
        <h4>
          React
        </h4>
        <p>
          A Javascript library for building user interfaces.
        </p>
      </div>
    </div>
  </div>
</div>
`;
